src/components/Navbar/Navbar.module.css

```diff
@@ -10,14 +9,22 @@
     width: 100%;
     top: 0;
     z-index: 2;
+    height: 6%;
```

Comment:
Generally not a good idea to rely on percentages for stuff like this. Use pixels or rem for what you want to achieve. % will scale based on viewport size and will often not be ideal